src/Link.js

```diff
@@ -4,6 +4,7 @@ import { color } from 'styled-system'
 import theme from './theme'
 
 const Link = styled.a`
+  cursor: pointer;
```

Comment:
Not opposed to this change, but when a link has an `href` (which it should) the cursor will be a pointer – *not* setting this style can sometimes be a good way to notice accessibility issues